doc/modules/mod_fast_auth_token.md

```diff
@@ -37,3 +37,18 @@ Set it to 0 to disable automatic rotation.
   validity_period.access = {value = 1, unit = "days"}
   validity_period.rotate_before_expire = {value = 0, unit = "days"}
 ```
+
+## 0-RTT Support
+
+For support, set `session_tickets = "stateless"` and `early_data = true` for
+the C2S listener on port 5223 (Legacy or Direct TLS).
```

Comment:
Please call it just "Direct TLS", we need to get rid of all mentions of "legacy" when it is the only sane default.